trufflescripts/utils/index.js

```diff
@@ -26,9 +26,18 @@ module.exports = (web3) => {
     increaseTimeBy(increaseBy)
   }
 
+  const makeSnapshot = () => web3.currentProvider.send({ jsonrpc: '2.0', method: 'evm_snapshot' }).result
+
+  const revertSnapshot = async (blockN = '0x01') => {
```

Comment:
Not a block number
`send` is synchronous, no need to `await`
if you specifically want async, use `sendAsync`